meshio/medit_io.py

```diff
@@ -118,13 +118,11 @@ def read_buffer(file):
 def write(filename, mesh):
     with open(filename, "wb") as fh:
         if mesh.points.dtype == c_float:
-            fh.write(b"MeshVersionFormatted 1\n")
-        elif mesh.points.dtype == c_double:
-            fh.write(b"MeshVersionFormatted 2\n")
+            version = 1
         else:
-            raise NotImplementedError(
-                "Unknown points.dtype '{}'.".format(mesh.points.dtype)
-            )
+            assert mesh.points.dtype == c_double
+            version = 2
+        fh.write(b"MeshVersionFormatted %d\n" % version)
```

Comment:
New-style `format` please.

Reply:
Hmm, no: `AttributeError: 'bytes' object has no attribute 'format'`.

The makeshift at https://stackoverflow.com/questions/15710515/python-3-bytes-formatting is reverting to old-style `%`.

Reply:
Nice. Best add a link to that question in the code.

Reply:
Oh, I see that there's another idiom already in use immediately following:
```
        dim = "\nDimension {}\n".format(d)
        fh.write(dim.encode("utf-8"))
```

I'll adopt that for consistency.

By the way, is all this `open(…, "wb")`, `.write(b"…")`, `.write(_.encode("utf-8"))` stuff still needed in Python 3?  I note that `meshio.medit_io` only deals with text MEDIT mesh thus far, not binary #295.